inference-engine/ie_bridges/c/samples/common/opencv_c_wraper/CMakeLists.txt

```diff
@@ -10,7 +10,7 @@ file(GLOB SOURCES ${CMAKE_CURRENT_SOURCE_DIR}/*.cpp)
 file(GLOB HEADERS ${CMAKE_CURRENT_SOURCE_DIR}/*.h)
 
 # create library
-add_library(${TARGET_NAME} SHARED ${HEADERS} ${SOURCES})
```

Comment:
this code is a part of release OpenVINO package where `BUILD_SHARED_LIBS` is not defined. I think we need to add explicit `BUILD_SHARED_LIBS=ON` inside https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/samples/CMakeLists.txt which works for both C and C++ samples.